src/mds_array_manipulation/mds_array_manipulation.py

```diff
@@ -1,3 +1,31 @@
+def search_array(arr, elem):
+    """
+    Search for elem in a numpy array.
+    Parameters
+    ----------
+    arr : numpy.array
+        The numpy array to search
+    elem : object to search for in the array
+    Returns
+    -------
+    int
+        Index of the first occurence of the element in the array, -1 if not found
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from mds_array_manipulation import search_array
```

Comment:
Maybe you can change the import statement
```suggestion
    >>> from mds_array_manipulation.mds_array_manipulation import search_array
```

Reply:
Agreed with the change suggested by Aishwarya120111